clientv3/integration/watch_test.go

```diff
@@ -211,6 +211,14 @@ func testWatchReconnRequest(t *testing.T, wctx *watchctx) {
 	stopc <- struct{}{}
 	<-donec
 
+	// spinning on dropping connections may trigger a leader election
+	// due to resource starvation; l-read to ensure the cluster is stable
+	ctx, cancel := context.WithTimeout(30 * time.Second)
+	if _, err := wctx.kv.Get(ctx, "_"); err != nil {
```

Comment:
this wont help. l-read would fail too. now we might drop proposals when there is a leader election :(